src/NuGet.Core/NuGet.Packaging/PackageCreation/Utility/NuGetFrameworkNameUtility.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.Versioning;
+
+using NuGet.Frameworks;
+
+namespace NuGet.Packaging
+{
+    public static class NuGetFrameworkNameUtility
```

Comment:
This class is used a few times within the same project/assembly, but also once from `NuGet.CommandLine`. Is there a way to refactor a little so it doesn't need to be used by `NuGet.CommandLine`, and change the class to `internal`?

`NuGetFrameworkNameUtility` doesn't seem like an API that should be public for customers using `NuGet.Packaging` in their own apps.

Reply:
I'm confused. Does this file already exist? https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Packaging/PackageCreation/Utility/FrameworkNameUtility.cs

Why does it appear as a new file in this PR?

Reply:
FrameworkNameUtility vs FrameworkNameUtility. 

I think we can combine them into 1.

Reply:
One returns FrameworkName, this one returns NuGetFramework.

Reply:
I've merged them into a single utility.